docs/framework-agnostic-plugins.md

```
@@ -0,0 +1,2581 @@
+  ViewComponent: View,
+  PreviewComponent: Preview,
+};
+```
+
+---
+
+## 外部プラグインの現状構造
+
+### 現在のディレクトリ構成（移行前）
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)